Hardcore.lua

```diff
@@ -709,10 +723,36 @@ function Hardcore:CHAT_MSG_ADDON(prefix, datastr, scope, sender)
 		-- Get the command
 		local command, data = string.split(COMM_COMMAND_DELIM, datastr)
 
+		if alert_msg_time[command] == nil then -- unknown command received
+			local debug_info = {command, data, sender}
+			table.insert(Hardcore_Settings.debug_log, debug_info)
+			alert_msg_time[command][sender] = time()
+			-- Display that someone is trying to send unknown messages; notifies mods to look at saved_vars and remove player from guild
+			if spam_alert_warning[sender] ~= nil then
+				Hardcore:Print("Receiving unknown messages from " .. sender .. ".")
+				spam_alert_warning[sender] = 1
+			end
+			return
+		end
+
+		if alert_msg_time[command][sender] and (time() - alert_msg_time[command][sender] < COMM_SPAM_THRESHOLD[command]) then
+			local debug_info = {command, data, sender}
+			table.insert(Hardcore_Settings.debug_log, debug_info)
+			alert_msg_time[command][sender] = time()
+			-- Display that someone is trying to send spam messages; notifies mods to look at saved_vars and remove player from guild
+			if spam_alert_warning[sender] ~= nil then
+				Hardcore:Print("Receiving spam from " .. sender .. ".")
```

Comment:
Maybe would should have a mod-mode? Similar to debug, but specific for this message so that normal users don't see that they are being attacked. Mods would see it and silently remove the person from the guild

Reply:
I like the idea of having a mod mode for this

Reply:
added `monitor`; enabled with `/hc monitor`, which will show display messages for weird chat comms